test/index.js

```diff
@@ -273,7 +296,8 @@ describe('restify-client tests', function () {
         });
     });
 
-    it('GH-778 GET jsonp', function (done) {
+
+    it.skip('GH-778 GET jsonp', function (done) {
```

Comment:
Shouldn't we remove this test?

Reply:
I wanted to add a TODO, but you're right - I'll remove and file an issue instead.